content/projects/gsoc.adoc

```diff
@@ -22,19 +22,38 @@ in completing their summer projects.
 
 == GSoC 2018
 
-* link:/projects/gsoc/gsoc2018-project-ideas[GSoC 2018 Project ideas]
-* link:/blog/2018/01/05/gsoc2018-call-for-mentors[Call for Mentors]
+Jenkins project participates in Google Summer of Code 2018.
+
+* link:https://summerofcode.withgoogle.com/organizations/5572716199936000/[Jenkins page on GSoC website]
 * link:/projects/gsoc/students[Information for students]
 * link:/projects/gsoc/mentors[Information for mentors]
+* link:/projects/gsoc/gsoc2018-project-ideas[GSoC 2018 Project ideas]
+* link:/blog/2018/01/06/gsoc2018-call-for-mentors[Call for Mentors]
```

Comment:
I wish we had a check for broken links in the PR builder (at least internal ones)